src/main.ts

```diff
@@ -29,8 +32,17 @@ import { getCommentMessage } from "#/main/modules/get_comment_message";
 export const _main_ = async () => {
   // Verify correct environment and request context
   requireEvent();
-  requirePullNumber();
+  requirePullNumber(); // No needed, remove it ?? 
```

Comment:
What does this function do?  Why do you think we no longer need it?

Reply:
// redundant, and if it's going to stay needs to return value to const pullNumber..
export const requirePullNumber = () => {
  const pullNumber = github.getPullNumber();

  if (!pullNumber) {
    throw new CriticalError(
      "Build does not have a PR number associated with it; quitting..."
    );
  }
return pullNumber;

// invoked in main.ts just after requirePullNumber()
 export const requirePr = async (): Promise<PR> => {
  const prNum = requirePullNumber();
  const pr = await github.getPullRequestFromNumber(prNum);

  if (pr.merged && process.env.NODE_ENV !== "development") {
    throw new CriticalError(`PR ${prNum} is already merged; quitting`);
  }
return pr;
};

Reply:
If it is truly redundant, we should probably remove it rather than just commenting on it.